src/components/BookItem.js

```diff
@@ -3,27 +3,55 @@ import { useDispatch } from 'react-redux';
 import PropTypes from 'prop-types';
 import { removeBook } from '../redux/books/books';
 
+// stylesheet
+import './BookItem.css';
+
 function BookItem({ book }) {
   const dispatch = useDispatch();
   // eslint-disable-next-line camelcase
   const { title, author, item_id } = book;
   return (
-    <div>
-      <h2>
-        {title}
-      </h2>
-      <h2>
-        {author}
-      </h2>
-      <button
-        type="button"
-        onClick={() => {
-          dispatch(removeBook(item_id));
-        }}
-      >
-        Remove
 
-      </button>
+    <div className="book-item-container">
+      <div className="title-section">
+        <div>
+          <span className="book-category">Nature</span>
+          <h2 className="book-title">
+            {title}
+          </h2>
+          <span className="book-author">
+            {author}
+          </span>
+        </div>
+        <div className="action-buttons">
+          <button type="button">Comments</button>
+          <button
+            type="button"
+            onClick={() => {
+              dispatch(removeBook(item_id));
+            }}
+          >
+            Remove
+          </button>
+          <button type="button">Edit</button>
+        </div>
+      </div>
+
+      <div className="progress-container">
+        <div className="progress-section">
+          <div className="circular-progress" />
+          <div>
+            <h3 className="circular-progress-percent">65%</h3>
+            <span className="progress-span">Completed</span>
+          </div>
+        </div>
+        <div className="progress-section-divider" />
+        <div className="current-chapter-section">
+          <span className="cur-chapter-title">CURRENT CHAPTER</span>
+          <h4 className="cur-chapter">Chapter 10</h4>
+          <button type="button" className="update-progress-btn">UPDATE PROGRESS</button>
+        </div>
+      </div>
     </div>
```

Comment:
- [ ] Kindly note that we are to style the app to match the design EXACTLY. Kindly improve your bookitem styling to match the design. Kindly use the correct font styling.

- [ ] Kindly move the `Update Progress` section on the right demarcation more towards the left to match to match the design.

  Link to the design: https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/sneak_peek_v2_0.md

  ![kisi](https://user-images.githubusercontent.com/54848363/230662572-d00a7913-0d7e-40e1-8fcc-6e410de91135.jpg)

Reply:
Ok well noted